cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -57,6 +57,17 @@ import           Ouroboros.Network.Protocol.LocalStateQuery.Type as LocalStateQu
 import qualified Shelley.Spec.Ledger.API.Protocol as Ledger
 import           Shelley.Spec.Ledger.Scripts ()
 
+-- These imports are used specifically for stake snapshots
+import           Shelley.Spec.Ledger.LedgerState hiding (LedgerState,_delegations)
+import           Shelley.Spec.Ledger.EpochBoundary
+import           Shelley.Spec.Ledger.Coin
+import           Shelley.Spec.Ledger.Keys (KeyHash(..))
+import           Data.Maybe
+import           Cardano.Ledger.Era
+
```

Comment:
The convention for this repo is that all imports from outside this repo *MUST* be explcit or qualified.

These imports should be distributed amongst the above imports.

Reply:
OK.  There's another Cardano.Ledger.Era import in Genesis.hs (not touched by me).  I'll fix that one as well